### ISSUE TITLE
The new green (#319795) doesn't meet accessibility contrast standards when used with white (the contrast ratio is only 3.5:1 and should be 4.5:1 unless you're at large text sizes).

### ISSUE THREAD

**Issue by dhicking** [tightenco/onramp#216]
This applies to both green on white and white on green.

**Comment by dhicking** [comment]
<img width="328" alt="Screenshot 2020-06-26 14 04 55" src="https://user-images.githubusercontent.com/828307/85887688-43f0b200-b7b6-11ea-917d-5abc1c31c7f0.png">

^ Screenshot from WAVE.
